packages/suite-desktop/src/libs/processes/CoinjoinProcess.ts

```diff
@@ -0,0 +1,40 @@
+import { BaseProcess, Status } from './BaseProcess';
+
+export class CoinjoinProcess extends BaseProcess {
+    constructor() {
+        super('coinjoin', 'WalletWasabi.WabiSabiClientLibrary', {
```

Comment:
Done.
I added default port that is basically always going to fail because when this class is initialized in `packages/suite-desktop/src/modules/coinjoin.ts` inside the `export const init: Module = ({ mainWindow }) => {` and modules are not loaded asynchronous so I thought it would be better idea to use `await getFreePort` everytime `onCreateInstance` is called. Because it is called asynchronously and that way we make sure we always get a free available port.